hexrd/ui/calibration/polar_plot.py

```diff
@@ -132,7 +132,7 @@ def write_image(self, filename='polar_image.npz'):
             if name not in HexrdConfig().visible_masks:
                 continue
 
-            data[f'mask_{name}'] = mask
+                data[f'mask_{name}'] = mask
```

Comment:
```suggestion
            data[f'mask_{name}'] = mask
```

I think this should be switched back to the previous indentation. Otherwise, this is unreachable code.

Reply:
Thanks, fixed.